RR.App/Controllers/HRIS/EmployeeDataController.cs

```diff
@@ -23,95 +25,40 @@ public EmployeeDataController(AuthorizeIdentity identity, IEmployeeDataService e
     [HttpGet]
     public async Task<IActionResult> GetEmployeeData([FromQuery] int id)
     {
-        try
-        {
-            if (_identity.Role is "SuperAdmin" or "Admin" or "Talent" or "Journey")
-            {
-                var getEmployeeData = await _employeeDataService.GetAllEmployeeData(id);
-                if (getEmployeeData == null) throw new Exception("Employee data not found");
-                return Ok(getEmployeeData);
-            }
-
-            if (id == _identity.EmployeeId)
-            {
-                var getEmployeeData = await _employeeDataService.GetAllEmployeeData(id);
-                if (getEmployeeData == null) throw new Exception("Employee data not found");
-                return Ok(getEmployeeData);
-            }
-            return NotFound("User data being accessed does not match user making the request.");
-        }
-        catch (Exception ex)
-        {
-            return NotFound(ex.Message);
-        }
+           if (_identity.Role is not ("SuperAdmin" or "Admin" or "Talent" or "Journey") && id != _identity.EmployeeId)
+           throw new CustomException("User data being accessed does not match user making the request.");
+      
+           var data = await _employeeDataService.GetAllEmployeeData(id);
+           return Ok(data);
```

Comment:
Please fix indentation. A short-cut is to press `CTRL + K+ D` when viewing the file in visual studio

```suggestion
        if (_identity.Role is not ("SuperAdmin" or "Admin" or "Talent" or "Journey") && id != _identity.EmployeeId)
            throw new CustomException("User data being accessed does not match user making the request.");

        var data = await _employeeDataService.GetAllEmployeeData(id);
        return Ok(data);
```